app/namespaces/test_adjustment/test_adjustment_controller.py

```diff
@@ -0,0 +1,66 @@
+import logging.config
+
+from flask_restplus import Resource, Namespace
+from flask import jsonify, make_response, request
+
+from app.utils import validate_request_input_against_schema
+from .test_adjustment_schema import TestAdjustmentSchema
+from .test_adjustment_service import TestAdjHandler
+
+
+test_adjustment_ns = Namespace('test_adjustment', description='Endpoints for updating airtable records via scripting.')
+logging.getLogger(__name__)
+
+
+@test_adjustment_ns.route('/get_adjusted', methods=['POST'])
+class Records(Resource):
+    @test_adjustment_ns.doc('An endpoint for getting all records from database with or without filters.')
+    def post(self):
+        # Convert input payload to json and throw error if it doesn't exist
+        data = request.get_json()
+        if not data:
+            return {"message": "No input payload provided"}, 400
+
+        # Log request info
+        logging.info("Endpoint Type: {type}, Endpoint Path: {path}, Arguments: {args}, Payload: {payload}".format(
+            type=request.environ['REQUEST_METHOD'],
+            path=request.environ['PATH_INFO'],
+            args=dict(request.args),
+            payload=data))
+
+        # Validate input payload
+        payload, status_code = validate_request_input_against_schema(data, TestAdjustmentSchema())
+        if status_code != 200:
+            # If there was an error with the input payload, return the error and 422 response
+            return make_response(payload, status_code)
+
+        # All of these params can be empty, in which case, our utility functions will just return all records
```

Comment:
Why return all records? Shouldn't it just return the record of interest?

When we chat, let's chat about what this endpoint is for, exactly. What is the payload - shouldn't it just be a dict of the inputs required for test adjustment - and the result - shouldn't it just be posterior prev, sens, and spec?